routes/applicationRoutes.js

```diff
@@ -91,13 +96,16 @@ router.put('/:id?approve', async (req, res) => {
 		return res.status(200).json();
 
 	} catch(error) {
-		switch (error.message) {
+		console.error(error.code);
+		switch (error.code) {
 		case 'E1005':
-			return res.status(400).json({ 'error': errorCodes['E1005'] }); // 'Could not get Content Creator application'
```

Comment:
@magnusklundgren I dont like that, we remove the ErrorCodes, because, In the frontend, each error code has its own Custom Portogeese Error text, Which is displayed with Toast. When we just send error.message,  the user will get notified with Axios.Error.etc
Bring back the Errorcodes.